app/Http/Controllers/GroupCreateController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Group;
+use App\GroupMember;
+
+use Illuminate\Http\Request;
+
+class GroupCreateController extends Controller
+{
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $req
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $req)
+    {
+        if (isset($req->name) &&  Group::where('name', '=', $req->name)->get()->isEmpty()) {
```

Comment:
Can you do the validation this way: 
Example:
$this->validate($req, [
    'title' => 'bail|required|unique:posts|max:255',
    'body' => 'required',
]);

to follow the convention we're doing